portfolio/src/main/webapp/about.js

```diff
@@ -0,0 +1,93 @@
+let answer = 0;
+
+function loadInTruth() {
+
+    let firstTruth = Math.floor(Math.random() * truths.length);
+    let secondTruth = Math.floor(Math.random() * truths.length);
+    while (firstTruth == secondTruth && truths.length != 1) {
+        secondTruth = Math.floor(Math.random() * truths.length);
+    }
+
+    let firstPlace = Math.floor(Math.random() * 3);
+    let secondPlace = Math.floor(Math.random() * 3);
+    while (firstPlace == secondPlace) {
+        secondPlace = Math.floor(Math.random() * 3);
+    }
+
+    let truthLocations = new Array(2);
+    truthLocations[0] = firstPlace;
+    truthLocations[1] = secondPlace;
+
+    if (firstPlace==0) {
+        document.getElementById("first_txt").innerHTML = truths[firstTruth][0];
+        document.getElementById("first_img").src = truths[firstTruth][1];
+    } else if (firstPlace==1) {
+        document.getElementById("second_txt").innerHTML = truths[firstTruth][0];
+        document.getElementById("second_img").src = truths[firstTruth][1];
+    } else if (firstPlace==2) {
+        document.getElementById("third_txt").innerHTML = truths[firstTruth][0];
+        document.getElementById("third_img").src = truths[firstTruth][1];
+    }
+
+    if (secondPlace == 0) {
+        document.getElementById("first_txt").innerHTML = truths[secondTruth][0];
+        document.getElementById("first_img").src = truths[secondTruth][1];
+    } else if (secondPlace==1) {
+        document.getElementById("second_txt").innerHTML = truths[secondTruth][0];
+        document.getElementById("second_img").src = truths[secondTruth][1];
+    } else if (secondPlace==2) {
+        document.getElementById("third_txt").innerHTML = truths[secondTruth][0];
+        document.getElementById("third_img").src = truths[secondTruth][1];
+    }
```

Comment:
This code is duplicated twice, so instead, make it a method where you pass in firstPlace and secondPlace, respectively.

Also, I might recommend using a switch statement: https://www.w3schools.com/js/js_switch.asp

switch(placeVariable) {
   case 0:
   case 1:
   case 2:
}

Reply:
Added load in fact function with switch statement to reduce reused code